src/node_interface.rs

```diff
@@ -28,6 +28,7 @@ pub enum MethodCall {
     Get { type_id: u64, name: NameType, },
     Post,
     Refresh { content: Box<Sendable>, },
+    PutResponse { destination: NameType, payload: Vec<u8> },
```

Comment:
Can we rename this though to 'SendOn { destination : NameType }`;  A PutResponse is only generated after conversion from a Put on Error or Reply;  So you can add an error with with payload Vec<u8> to have this payload included.